modules/templates/proc-creating-templates.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * admin/assembly-admin-templates.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="proc-creating-templates_{context}"]
+= Creating a template by using the Template Editor
+
+You can create a template by using the Template Editor. You can access the Template Editor by opening the URL `\https://<RHDP_INSTANCE/create/edit`.
```

Comment:
If we use UI, maybe also give the option to navigate to the Template Editor from UI?

Click _Create..._ from the left menu and then click the three dots in the top right corner and select _Template editor_

Reply:
Added